app/styles/components/addon-tabs.css

```diff
@@ -12,6 +12,7 @@
 }
 
 .addon-tabs--tab {
+  align-content: center;
```

Comment:
```suggestion
  align-items: center;
```

With `align-content` it looks like this: 

![Screenshot 2020-02-17 at 16 49 04](https://user-images.githubusercontent.com/594890/74672606-8faaa380-51a5-11ea-9ec1-950736c45557.png)

I assume that is no-change from what it looked like before? 

With `align-items` it looks like this: 

![Screenshot 2020-02-17 at 16 49 42](https://user-images.githubusercontent.com/594890/74672634-a224dd00-51a5-11ea-8e64-a86e735384c8.png)

Which I assume fixes the issue?